TaskModules/powershell/TaskModuleSqlUtility/SqlQueryOnTargetMachines.ps1

```diff
@@ -106,7 +68,8 @@ function Execute-SqlQueryDeployment
             }
         }
 
-        Invoke-Sqlcmd @spaltArguments
+        Write-Verbose "Invoke-SqlCmd arguments : $spaltArgumentsToLogJson  $additionalArguments"
+        Invoke-Expression "Invoke-SqlCmd @spaltArguments $additionalArguments"
```

Comment:
1) Can you pass additional arguments as named parameter.
2) Are there L0 that needs update?